### PR TITLE
Update docs for multi-instance setup and introduce CSS_CUSTOM_FILES variable

### PR DESCRIPTION
**Pull request by tsauerwein**
The first commit introduces the variable `CSS_CUSTOM_FILES` which allows to include instance-specific CSS files into the CSS build, so that CSS styles can be overridden in an instance. The instance-specific CSS files will be minified together will the other files.

Usage: e.g. `instance1.mk`

    INSTANCE_ID = instance1
    INSTANCE = instance1
    VARS_FILE = vars_instance1.yaml
    CSS_CUSTOM_FILES = project/static/css/proj-instance1.css

    include project.mk

The second commit updates the docs.

### CODE REVIEW

File: doc/integrator/create_application.rst [hunk 1]
```diff
@@ -319,17 +319,17 @@ the project.
...
 * One make file for each instance ``<instance>.mk``.
-* One Makefile generator for each developer and server ``<user>.mk.jinja``.
+* One Makefile generator for each developer and server ``<user>.mk``.
 * One additional CSS file for each instance ``<package>/static/css/proj_<instance>.css``.
 
 Create the project
 ..................
 
-1. Configure the instances in thee ``vars_<package>.yaml`` as follow:
+1. Configure the instances in ``vars_<project>.yaml`` as follows:
```
Comment (sbrunner) [inline doc/integrator/create_application.rst]: It should be package, it's the above docs who is wrong !

File: doc/integrator/create_application.rst [hunk 2]
```diff
@@ -319,17 +319,17 @@ the project.
...
 
 * One folder per instance ``mapfile/<instance>``.
-* One configuration file for all the project ``vars_<project>.yaml``.
+* One configuration file for the project ``vars_<project>.yaml``.
 * One configuration file for each instance ``vars_<instance>.yaml``.
-* One make file for all the project ``<project>.mk``.
+* One make file for the project ``<project>.mk``.
 * One make file for each instance ``<instance>.mk``.
-* One Makefile generator for each developer and server ``<user>.mk.jinja``.
+* One Makefile generator for each developer and server ``<user>.mk``.
```
Comment (sbrunner) [inline doc/integrator/create_application.rst]: Should be a jinja file !

File: doc/integrator/create_application.rst [hunk 3]
```diff
@@ -355,66 +353,63 @@ Create the project
 
     .. code:: make
 
+        INSTANCE_ID = <instance>
```
Comment (sbrunner) [inline doc/integrator/create_application.rst]: Why do we need this ?

File: doc/integrator/create_application.rst [hunk 4]
```diff
@@ -355,66 +353,63 @@ Create the project
...
         INSTANCE = <instance>
-        VARS_FILE = vars_$(INSTANCE).yaml
```
Comment (sbrunner) [inline doc/integrator/create_application.rst]: Why change this ?

File: doc/integrator/create_application.rst [hunk 5]
```diff
@@ -355,66 +353,63 @@ Create the project
...
-        include <package>.mk
```
Comment (sbrunner) [inline doc/integrator/create_application.rst]: This change is wrong !

File: doc/integrator/create_application.rst (hunk 4 above)
Comment (sbrunner) [inline doc/integrator/create_application.rst]: Finally all I see from this point is wrong ...
I relay don't know you try to do ...

**Comment by tsauerwein** [comment]
Apparently everything wrong, closing.

**Pull request closed without merging**